wrappers/wrapper.rb

```diff
@@ -753,6 +809,46 @@ def simplify_constraints(vrp)
       vrp
     end
 
+    def empty_result(solver, vrp, unassigned_reason = nil)
+      {
+        solvers: [solver],
+        cost: nil,
+        costs: Models::Costs.new({}),
+        iterations: nil,
+        routes: vrp.vehicles.collect{ |vehicle| { vehicle_id: vehicle.id, activities: [] } },
+        unassigned: (vrp.services.collect{ |service|
+          {
+            service_id: service.id,
+            type: service.type.to_s,
+            point_id: service.activity.point_id,
+            detail: build_detail(service, service.activity, service.activity.point, nil, nil, nil),
+            reason: unassigned_reason
+          }
+        }) + (vrp.shipments.collect{ |shipment|
+          [{
+            shipment_id: shipment.id,
+            type: 'pickup',
+            point_id: shipment.pickup.point_id,
+            detail: build_detail(shipment, shipment.pickup, shipment.pickup.point, nil, nil, nil),
+            reason: unassigned_reason
+          }] << {
+            shipment_id: shipment.id,
+            type: 'delivery',
+            point_id: shipment.delivery.point_id,
+            detail: build_detail(shipment, shipment.delivery, shipment.delivery.point, nil, nil, nil, true),
+            reason: unassigned_reason
+          }
```

Comment:
```suggestion
          }.delete_if{ |_k, v| !v }
```
to get rid of `reason: nil` in case `unassigned_reason` is not suplied